nims-notes-things/src/components/Form.js

```diff
@@ -58,53 +51,57 @@ const Form = (props) => {
         ...formData,
         id: uniqueId,
       };
-      const notesCopy = [...notes];
-      notesCopy.push(formDataCopy);
-      dispatch(setNote(notesCopy));
+      dispatch(setNote(formDataCopy));
     }
   };
 
-  const handleOnBlur = (value, key) => {
-    setFormEmpty((previousState) => {
-      return {
-        ...previousState,
-        [key]: value === "",
-      };
-    });
+  const handleOnBlur = (key) => {
+    emptyData.current[key].innerHTML =
+      formData[key] === "" ? `${key} required` : "";
   };
 
   return (
     <StyledForm onSubmit={handleOnSubmit}>
       <StyledDiv>
         <InnerForm>
-          {empty.name && <ErrorMessage>Name field is required</ErrorMessage>}
+          {notes.length === 0 && <p> Please Enter your Note</p>}
```

Comment:
The header of the **form** is only display when there are no notes, is that normal?

Reply:
Sorry this wasn't the header ... it was meant to be the message for when there are no notes submitted .... i've repositioned it and have also re-worded it